build-config/conf/onie-release

```diff
@@ -0,0 +1 @@
+2017.08.01
```

Comment:
A PR for the master branch will never include this file.  This file in only present in release branches.